cql/DashboardLibrary.cql

```diff
@@ -993,6 +1000,32 @@ define function FormatReports(DrList List<DiagnosticReport>, ObsList List<Observ
     edited: HasBeenLocallyEdited(D)
   }
 
+/* This will need to be refactored to support returning all results for a Diagnostic Report, not
+* just the highest ranked interpretation. The longValue does return the display attribute for
+* all conclusion codes included in the diagnostic report, but the UI will need to see individual
+* values/labels and grades.
+*/
```

Comment:
```suggestion
/* The FormatBiopsyReports function has the same structure as the FormatReports function above,
* but allows for a code to passed in so that DiagnosticReports can be recognized as 
* biopsy reports if their associated observation(s) have a biopsy code
*/
```
I had originally just copied the `FormatReports` function entirely, including the comment. But it may be helpful to consider changing the comment to something like this, to make it more easily understandable why there are separate functions that appear to do similar things.

Reply:
Updated comment.